### PR TITLE
Add support for parcel items

### LINKED ISSUE #1579: Add support for parcel items
Our SDK should support the new parcel items API.

### PR DESCRIPTION
**Pull request by acbeni**
Close #1579

### CODE REVIEW

File: commercetools-models/src/main/java/io/sphere/sdk/orders/ParcelImpl.java [hunk 1]
```diff
@@ -1,43 +0,0 @@
-package io.sphere.sdk.orders;
-
-import com.fasterxml.jackson.annotation.JsonCreator;
-import io.sphere.sdk.models.Base;
-
-import javax.annotation.Nullable;
-import java.time.ZonedDateTime;
-
-final class ParcelImpl extends Base implements Parcel {
```
Comment (katmatt) [inline commercetools-models/src/main/java/io/sphere/sdk/orders/ParcelImpl.java]: Great that you removed this class with a generated one 👍

File: commercetools-models/src/main/java/io/sphere/sdk/orders/ParcelMeasurementsImpl.java [hunk 2]
```diff
@@ -1,39 +0,0 @@
-package io.sphere.sdk.orders;
-
-import com.fasterxml.jackson.annotation.JsonCreator;
-import io.sphere.sdk.models.Base;
-
-final class ParcelMeasurementsImpl extends Base implements ParcelMeasurements {
```
Comment (katmatt) [inline commercetools-models/src/main/java/io/sphere/sdk/orders/ParcelMeasurementsImpl.java]: Great that you removed this class with a generated one 👍

**Comment by acbeni** [comment]
@katmatt added tests, don,t know why the commit doesn't show up here, but to view the added test check `OrderUpdateCommandIntegrationTest#testDeliveriesAndParcels()`

File: commercetools-models/src/test/java/io/sphere/sdk/orders/commands/OrderUpdateCommandIntegrationTest.java [hunk 3]
```diff
@@ -534,4 +535,27 @@ public void locale() {
             return updatedOrder;
         });
     }
+
+    @Test
+    public void testDeliveriesAndParcels() {
```
Comment (katmatt) [inline commercetools-models/src/test/java/io/sphere/sdk/orders/commands/OrderUpdateCommandIntegrationTest.java]: Thanks for adding this test 👍

**Comment by katmatt** [comment]
Looks good to me 👍

**Pull request merged**